Security/SessionDownloadStrategy.php

```diff
@@ -12,27 +12,18 @@
 namespace Sonata\MediaBundle\Security;
 
 use Sonata\MediaBundle\Model\MediaInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
 /**
- * Class SessionDownloadStrategy.
+ * Download strategy with time per Session.
```

Comment:
Remove because I don't understand what is meant by "with time per session" , I still have to look at the code to understand.